src/lang/internals/sclang-io.js

```diff
@@ -33,70 +36,83 @@ export const STATES = {
  */
 export class SclangIO extends EventEmitter {
   states: Object;
-  responseCollectors: Object;
-  capturing: Object;
```

Comment:
`responseCollectors` and `capturing` is a dict storing responses. There may be more than one active at any time.  They can post overlapping messages to STDOUT.